integration-testing/test/conftest.py

```diff
@@ -99,7 +106,7 @@ def validators_data(config):
 
 
 @pytest.yield_fixture(scope='session')
-def docker_client_session():
+def docker_client_session() -> Iterator["DockerClient"]:
```

Comment:
Iterator is too vague here, we would need a specific type for fixtures

Reply:
See [the link ](https://github.com/python/mypy/issues/4709)

Maybe we can wrap these Iterator into one type. I give it a try.